### PR TITLE
Grace/discussion view

### LINKED ISSUE #12: Static Mock Up of Discussion Forum Page
Set CSS styles for for discussion forum page, so that we can easily add react components and backend on top of your CSS.

### PR DESCRIPTION
**Pull request by gracesnoh**
Fix #12 : Discussion Static View
screen shot 2016-10-10 at 1 28 45 pm

### CODE REVIEW

**Comment by eric-ls** [comment]
add a screenshot of the features (if it's frontend) if possible :)

File: app/assets/stylesheets/components/_answer.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.answer-container {
```
Comment (eric-ls) [inline app/assets/stylesheets/components/_answer.scss:1]: nit: reduntant style?

File: app/assets/stylesheets/components/_forms.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+textarea {
+  box-sizing: border-box;
```
Comment (eric-ls) [inline app/assets/stylesheets/components/_forms.scss:2]: adding `appearance: none` will remove browser-specific styles (check out more [here](https://css-tricks.com/almanac/properties/a/appearance/))

**Review by eric-ls — CHANGES_REQUESTED** [review]
👍 small changes, but great job!

File: app/assets/stylesheets/components/_forms.scss (hunk 2 above)
Comment (gracesnoh) [reply app/assets/stylesheets/components/_forms.scss:2]: will do that in a separate branch when i add mixins!

**Pull request merged**